drivers/at24cxxx/include/at24cxxx_defines.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * @ingroup     drivers_at24cxxx
+ * @brief       Constants for AT24CXXX EEPROM devices.
+ * @{
+ *
+ * @file
+ * @brief       Constants for AT24CXXX EEPROM devices.
+ *
+ * @author      Fabian Hüßler <fabian.huessler@ovgu.de>
+ */
+
+#ifndef AT24CXXX_DEFINES_H
+#define AT24CXXX_DEFINES_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    AT24CXXX constants
+ * @{
+ */
+
+/** 14 bit for AT24C128, 15 bit for AT24C256 */
+#define AT24CXXX_ADDR_LEN          (2UL)
+/** page size */
+#define AT24CXXX_PAGE_SIZE         (64UL)
+/** number of poll attempts */
+#define AT24CXXX_MAX_POLLS         (6UL)
+/** delay between two poll attempts */
+#define AT24CXXX_POLL_DELAY_MS     (1UL)
```

Comment:
might as well make this `AT24CXXX_POLL_DELAY_US` and remove that multiplication with `US_PER_MS` on each call to ease the burden on the reader.

Reply:
Ok